bot/exts/utils/snekbox.py

```diff
@@ -318,6 +397,9 @@ async def continue_job(
                     timeout=10
                 )
 
+                if self.jobs[ctx.author.id][ctx.message.id]["bot_response_message_id"] != response.id:
+                    return None, None
+
```

Comment:
The purpose of this check is unclear. Can you leave an explanatory comment?

Reply:
https://github.com/python-discord/bot/pull/2218/commits/c7d364d0be77bce965ceba16ceea36063ced9998